src/diffusers/configuration_utils.py

```diff
@@ -336,7 +346,6 @@ def load_config(
                     subfolder=subfolder,
                     revision=revision,
                 )
```

Comment:
We pass revision here and never pass commit hash so it will always still fall through to hf_hub_download. Is that what's intended?

Reply:
yeah since we always need to sent a HEAD to at least one file to get the commit hash, so here we assume that the config is **always** downloaded before the model files.